packages/db-mongodb/src/types.ts

```diff
@@ -110,5 +111,65 @@ export type FieldToSchemaMap<TSchema> = {
   upload: FieldGeneratorFunction<TSchema, UploadField>
 }
 
-export type MigrateUpArgs = { payload: Payload; req: PayloadRequest }
-export type MigrateDownArgs = { payload: Payload; req: PayloadRequest }
+export type MigrateUpArgs = {
```

Comment:
I'm unsure why we didn't just have one `MigrateArgs` instead of MigrateUpArgs/MigrateDownArgs since `up` and `down` as they're the exact same. Too late now!